packages/server/src/seeder/utils.ts

```diff
@@ -163,23 +160,43 @@ export async function createArticlesPerUser(email: string, numberOfArticles: num
     // use the function generateFakeArticle to generate fake article
     // initially article should be created in draft form ( isPublished status set to false )
 
+    const prisma = PrismaClientSingleton.prisma;
+
+    const user = await prisma.user.findUnique({
+        where: {
+            email: email,
+        }
+    })
+
+    if (!user) {
+        return;
+    }
+
+
     for (let i = 0; i < numberOfArticles; i++) {
         const fakeArticle = await generateFakeArticle();
-        axios.post('/api/user/article', {
-            title: fakeArticle?.title,
-            subtitle: fakeArticle?.subtitle,
-            htmlContent: fakeArticle?.htmlContent,
-            markdownContent: fakeArticle?.markdownContent
-        }
-        )
-        .then(function (response) {
-            console.log("RESPONSE TIME:")
-            console.log(response);
-        })
-        .catch(function (error) {
-            console.log("ERROR")
-            console.log(error);
-        });
+
+        const title = fakeArticle!.title;
+        const subTitle = fakeArticle!.subtitle;
+        const htmlContent = fakeArticle!.htmlContent;
+        const markdownContent = fakeArticle!.markdownContent;
+
+        await prisma.user.update({
```

Comment:
It is fine but if you could use Promise.all it would speed up the things.

Reply:
promise.all when getting the values from fakeArticle?